packages/grafana-flamegraph/src/FlameGraph/rendering.ts

```diff
@@ -277,14 +337,24 @@ export function walkTree(
         collapsedItemRendered = item;
       }
 
-      renderFunc(item, barX, barY, width, height, label, muted);
+      renderObjects.push({
+        item,
+        x: barX,
+        y: barY,
+        width,
+        height,
+        label,
+        muted,
+      });
```

Comment:
this makes me nervous. it will allocate possibly 200k objects on every resize/redraw?